SharkCage/CageManager/CageManager.cpp

```diff
@@ -1,21 +1,28 @@
 #include "stdafx.h"
-#define WIN32_LEAN_AND_MEAN
 
 #include "../CageService/NetworkManager.h"
 
-#include <Windows.h>
+#define byte win_byte_override
```

Comment:
defines should be all uppercase

Reply:
If I don't use the define I get the following error a few hundred times:
```
1>c:\program files (x86)\windows kits\10\include\10.0.16299.0\shared\rpcndr.h(192): error C2872: 'byte': ambiguous symbol
1>c:\program files (x86)\windows kits\10\include\10.0.16299.0\shared\rpcndr.h(191): note: could be 'unsigned char byte'
1>c:\program files (x86)\microsoft visual studio\2017\enterprise\vc\tools\msvc\14.13.26128\include\cstddef(22): note: or       'std::byte'
```

Reply:
Hm, it seems this is needed by `GDI+`: https://stackoverflow.com/questions/45957830/gdipluspath-throws-ambiguous-byte-for-cstddef-and-rpcndr-h